### PR TITLE
feat: Additional QR code feature

### PR DESCRIPTION
**Pull request by uy-seng**
- display user name when their qr code is scanned
- add timestamp to backend when qr code is scanned

### CODE REVIEW

File: pages/admin/scan/index.tsx [hunk 1]
```diff
@@ -373,7 +391,16 @@ export default function Admin() {
...
+                          <p>{success ?? 'Unexpected error!'}</p>
+                          {scannedUserInfo && (
+                            <>
+                              <p>
+                                Name: {scannedUserInfo.user.firstName}{' '}
+                                {scannedUserInfo.user.lastName}
+                              </p>
+                            </>
+                          )}
+                          <p>{success ?? 'Unexpected error!'}</p>
```
Comment (DedsecKnight) [inline pages/admin/scan/index.tsx]: I believe this line is duplicated.

**Review by DedsecKnight — CHANGES_REQUESTED** [review]

File: pages/admin/scan/index.tsx (hunk 1 above)
Comment (uy-seng) [reply pages/admin/scan/index.tsx]: i removed the duplicated line

**Review by DedsecKnight — APPROVED** [review]
LGTM :+1:

**Review by raywa04 — APPROVED** [review]
changes seem good to me :)

**Pull request merged**